File: server/src/utils/enums.py
import enum


class BoardValue(enum.Enum):
    EMPTY = " "
    X = "X"
    O = "O"

class GameStatus(enum.Enum):
    INITIAL = "INITIAL"
    IN_PROGRESS = "IN_PROGRESS"
    PENDING = "PENDING"
    DONE = "DONE"

File: front/src/utils/enums.ts
enum Events {
    INIT_PARAMS = "init_params",
    GAME_STATE = "game_state"
};

enum GameStatus {
    INITIAL = "INITIAL",
    IN_PROGRESS = "IN_PROGRESS",
    PENDING = "PENDING",
    DONE = "DONE"
};

export {
    Events,
    GameStatus
};